test/unit/response.test.js

```diff
@@ -21,6 +21,33 @@ describe('Response', function () {
             expect(response).to.be.ok();
         });
 
+        it('should handle the absence of Buffer gracefully', function () {
+            var response,
+                originalBuffer = Buffer;
+
+            delete global.Buffer;
+            expect(function () {
+                response = new Response({ stream: originalBuffer('random').toJSON() });
```

Comment:
Can you move the value assigned to `stream` up as a variable?

It would make more sense to read the test as 

1. Create a value as `Buffer('random').toJSON()`
2. Remove `Buffer`
3. Make sure `new Response()` works without `Buffer`
4. Restore `Buffer`

Right now, removing the `Buffer` and accessing it again looks weird.